src/plugin/fieldViews/RichTextFieldView.ts

```diff
@@ -91,6 +93,8 @@ export const createFlatRichTextField = ({
     validators,
     placeholder,
     marks,
+    attrs,
+    absentOnEmpty,
```

Comment:
Added as this wasn't being passed through, and will be inevitably needed at some point!